fastlane/Fastfile

```diff
@@ -291,16 +301,29 @@ end
       locales: GLOTPRESS_TO_LPROJ_APP_LOCALE_CODES,
       download_dir: File.join(PROJECT_ROOT_FOLDER, 'Simplenote')
     )
-    sanitize_appstore_keywords()
-    ios_lint_localizations(input_dir: 'Simplenote', allow_retry: true)
-    ios_bump_version_beta()
+    sanitize_appstore_keywords
+    ios_lint_localizations(
+      input_dir: 'Simplenote',
+      allow_retry: true
+    )
+    ios_bump_version_beta
 
     # Wrap up
     version = ios_get_app_version(public_version_xcconfig_file: VERSION_FILE_PATH)
-    removebranchprotection(repository:GHHELPER_REPO, branch: "release/#{version}")
-    setfrozentag(repository:GHHELPER_REPO, milestone: version, freeze: false)
-    create_new_milestone(repository:GHHELPER_REPO)
-    close_milestone(repository:GHHELPER_REPO, milestone: version)
+    removebranchprotection(
```

Comment:
Similar here, `removebranchprotection` action has been renamed `remove_branch_protection` since `release-toolkit` v`9.1.0`; to keep in mind for subsequent PR?

Reply:
Fixed in https://github.com/Automattic/simplenote-ios/commit/0a760bd4f8d0d4787f1cbc523f1de13e8c0d3abd